.gitignore

```diff
@@ -12,6 +12,7 @@ CVS
 *.swp
 .DS_Store
 ._*
+out????/
```

Comment:
Added folder auto generated to deploy in gh-pages, this folder every created with random numbers